### PR TITLE
Avoid chmod

### PR DESCRIPTION
**Pull request by Menda**

### CODE REVIEW

File: service_builder/templates/docker/Dockerfile [hunk 1]
```diff
@@ -12,4 +12,4 @@ ENV environment=${environment}
 RUN echo "Environment: $environment"
 
 EXPOSE 8080
-ENTRYPOINT ["/code/docker-entrypoint.sh"]
+ENTRYPOINT ["bash", "/code/docker-entrypoint.sh"]
```
Comment (bogdangi) [inline service_builder/templates/docker/Dockerfile]: I would recommend to add additional step in `Dockerfile` to make the entrypoint executable.
```
RUN chmod +x /code/docker-entrypoint.sh
```
Or  we can add permission modification step in code to accept permissions for the file https://github.com/Humanitec/django-service-wizard/blob/e360903ccf98c7dc64de46853ef2c84d12af3682/service_builder/utils.py#L81,
and then we will do follow:
```
https://github.com/Humanitec/django-service-wizard/blob/e360903ccf98c7dc64de46853ef2c84d12af3682/service_builder/utils.py#L81
and copy state from the template fie like this:
```
import os
import stat

st = os.stat('docker-entrypoint.sh')
os.chmod('docker-entrypoint.sh', st.st_mode | stat.S_IEXEC)
```

File: service_builder/setup_service.py [hunk 2]
```diff
@@ -182,4 +182,7 @@ def setup():
     if is_answer_yes:
         _configure_drone_ci()
 
+    PrettyPrint.msg_blue(
```
Comment (bogdangi) [inline service_builder/setup_service.py]: This code is not tested

Comment (Menda) [reply service_builder/setup_service.py]: Done, added a test

Comment (bogdangi) [reply service_builder/setup_service.py]: 👍

File: service_builder/templates/docker/Dockerfile (hunk 1 above)
Comment (bogdangi) [reply service_builder/templates/docker/Dockerfile]: ok.

**Pull request merged**